src/project-2.js

```diff
@@ -1,41 +1,53 @@
 // Do not change any of the function names
 
 const getBiggest = (x, y) => {
+  return x > y ? x : y;
```

Comment:
Good use of a ternary operator.  You could do this too: `const getBiggest = (x, y) => x > y ? x : y;`